server/src/entities/test.ts

```diff
@@ -0,0 +1,7 @@
+export type TestResults = {
```

Comment:
I would name the type `TestResult` (singular) and update the file name to match in snake case `test_result`

```suggestion
export type TestResult = {
```